be/src/Unic.UrlMapper2/code/Definitions/Constants.cs

```diff
@@ -53,5 +55,10 @@ public struct Settings
 
             public const string OriginalUrlHeaderForJssProcessor = "UrlMapper2.OriginalUrlHeaderForJssProcessor";
         }
+
+        public struct RegexExpressions
```

Comment:
**nitpick:** As the `ex` in regex already stands for `expression`, maybe we can call the struct just `RegularExpressions` ?

Reply:
yep :) right